refactor: 🚚 rename from batches to staging (#1762)
seedcase-project/seedcase-sprout: docs/design/architecture/naming.qmd

  | Object             | Description                                                                                                                                                         |
  | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
  | package            | A data package that contains a collection of related data resources and properties.                                                                                 |

>   | resource           | A single data file within a package, along with its properties files.                                                                                               |

Reviewer: ```suggestion
  | resource           | A single data file within a package, along with its properties.                                                                                               |
```

  | observational unit | The unit of observation, meaning the data associated with a specific entity at a specific time, e.g. a person who had their weight measured on their 30th birthday. |

  : General objects used throughout Sprout.

  | File or folder | Description                                                                                                                                                        |
  | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
  | dir            | A shortened name for folder or directory.                                                                                                                          |
  | packages       | The folder that contains all packages. *Only for server environments*.                                                                                             |
  | resources      | The folder that contains all resource files in a package.                                                                                                          |

>   | staging        | The folder that contains the staging data files for a data resource. These are files that were timestamped when first added (or updated) as raw files.             |

Reviewer 2: ```suggestion
  | staging        | The folder that contains the staging data files for a data resource. These files are processed from raw data.             |
```
Or something that explains the relationship between staging and raw.
